s2fft/transforms/otf_recursions.py

```diff
@@ -80,7 +80,14 @@ def inverse_latitudinal_step(
     half_slices = [el + mm + 1, el - mm + 1]
 
     if precomps is None:
-        precomps = generate_precomputes(L, -mm, sampling, nside, L_lower)
+        precomps = generate_precomputes(
+            L=L,
+            spin=-mm,
+            sampling=sampling,
+            nside=nside,
+            forward=False,
+            L_lower=L_lower,
+        )
```

Comment:
This is same change as fix in #256 to resolve #255. Applied here as otherwise test in `tests/test_signal_generator.py::test_generate_flm` was failing with `L_lower != 0`.